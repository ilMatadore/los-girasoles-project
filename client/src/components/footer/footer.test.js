import React from 'react';
import { shallow } from 'enzyme';
import Footer from './footer.components';

it('Expect to render Card component', () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
})