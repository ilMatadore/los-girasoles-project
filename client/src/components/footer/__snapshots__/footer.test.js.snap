// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Card component 1`] = `
<div
  className="footer-container"
>
  <div
    className="footer-links-container"
  >
    <div>
      <Link
        className="OptionLink"
        to="/quienes"
      >
        Quienes somos
      </Link>
    </div>
    <div>
      <Link
        className="OptionLink"
        to="/"
      >
        Productos
      </Link>
    </div>
    <div>
      <Link
        className="OptionLink"
        to="/contact"
      >
        Contacto
      </Link>
    </div>
  </div>
  <div
    className="footer-copyright"
  >
    <p>
      Copyright Â© Granja Los Girasoles 
      2023
    </p>
  </div>
</div>
`;
